### DISCUSSION TITLE
Upgraded to v13 but banner with avaibale upgrade to v13 still shows

### DISCUSSION THREAD

**Discussion by adrnag** [gravitational/teleport#31792] [Q&A] [answered]
Hello,

I have upgraded teleport to v13 from v12, but the banner with the available upgrade to latest version v13 is still showing up when logging in to the web interface.

Is this intentional, how can I fix this?

The way I upgraded was by editing the teleport.list for apt source and modifying from v12 to v13, then just used apt to install the upgrade. I hope this is the way, because I could not find any docs explaining how to upgrade.

~# tctl status
Cluster      teleport                                                  
Version      13.3.8    

Thank you!

**Comment by webvictim** [comment, marked as answer]
Yes, that method is correct.

Usually the banner will disappear within a few hours after your auth servers are upgraded. Did you restart the Teleport service after upgrading with `sudo systemctl restart teleport`?

**Comment by adrnag** [comment]
@webvictim yes, I have restarted. It is gone now. Thank you!
